libs/types/src/election.ts

```diff
@@ -98,14 +98,14 @@ export const DistrictsSchema = z
   });
 
 // Candidates
-export type WriteInId = Id;
-export const WriteInIdSchema: z.ZodSchema<WriteInId> = z
+export type WriteInId = `__write-in` | `__write-in-${string}`;
```

Comment:
I think we also do some of the form `write-in__<candidate name>` for BMD ballots where we know the text of the write in. i found this in services/scan/src/build_cast_vote_record.test.tsx